apps/site-projects/src/components/modules/Projects/Project/Project.js

```diff
@@ -40,26 +40,27 @@ const Project = ({ project, theme }) => {
     <Wrapper>
       <div id="background" />
       <HeroSection
-        projectName={project.title}
-        projectCatchPhrase={project.catchPhrase}
-        heroImage={project.heroImage}
+        projectName={project.attributes.title}
+        projectCatchPhrase={project.attributes.catchPhrase}
```

Comment:
If we are using the `?` operator on the line below, it would likely be a good idea to extract this to a constant outside the `return` statement. E.g.

`const attributes = project?.attributes;`

then you could pass it into the component like:

`projectCatchPhrase={attributes?.catchPhrase}`

Reply:
did this, thanks for the suggestion.